docusaurus.config.js

```diff
@@ -1,126 +1,126 @@
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
-  title: 'OpenSauced',
-  tagline: 'The path to your next Open Source contribution',
-  url: 'https://docs.opensauced.pizza',
-  baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
-  organizationName: 'open-sauced', // Usually your GitHub org/user name.
-  projectName: 'docs.opensauced.pizza', // Usually your repo name.
+  title: "OpenSauced",
+  tagline: "The path to your next Open Source contribution",
+  url: "https://docs.opensauced.pizza",
+  baseUrl: "/",
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+  favicon: "img/favicon.ico",
+  organizationName: "open-sauced", // Usually your GitHub org/user name.
+  projectName: "docs.opensauced.pizza", // Usually your repo name.
   trailingSlash: true,
   themeConfig: {
     algolia: {
-      appId: 'bh4d9od16a',
-      apiKey: '1691a4a5b06166984397aa30153f663c',
-      indexName: 'opensauced',
-      algoliaOptions: { 'facetFilters': ["type:$TYPE"] },
+      appId: "RH1WG3CUDK",
+      apiKey: "ea723aba7d69f2f36feac9292f2d2fe9",
```

Comment:
@OgDev-01 just double checking, this is a public API key that's safe to expose here, right?